## Trainers

- Miguel Andrade
- Aidan Budd
- Holger Dinkel
- Nadya Doncheva
- Zsuzsanna Dosztányi
- Toby Gibson
- Lars Juhl Jensen
- Tamás Korcsmáros
- Scooter Morris
- Nicky Mulder
- Marco Punta
- Vicky Schneider
- Ora Schueler-Furman
- Allegra Via
- Natasha Wood

## Topics

**prediction and analysis of protein modules**

-  globular; Marco, Allegra
-  IUP (ELMs, repeats, etc.); Zsuzsa, Marco, Miguel, Ora

**programic access of bioinformatics resources**

- introduction to Linux command line; Holger, Allegra
- REST interfaces (STRING, Uniprot etc.); Holger, Lars

**network approaches**

- Cytoscape; Scooter, Nadya
- String; Lars

**activities, networking**; Aidan, Tamas, Vicky

**introduction to PPI resources and databases**; Nicky, Aidan, Vicky, Marco, Lars, Scooter, Nadya, everyone, I guess

**overviews and keynotes**; Toby, Lars, Ora, Tamas

- electronic health records; Lars
- modelling linear motif-domain interactions; Ora
- modular protein architecture & construction of cell regulatory systems; Toby

**predicting, analysing, visualising protein and complex 3D structures**; Scooter, Allegra

**sequence alignments**; Toby, Marco, Scooter, Allegra

## Planned changes to the program

- Change the order of the 'story' - introducing networks early on. We know you can tell the story from both direction, let's try it from networks down this time
    - we'll try listing the 'dependencies' of the different topic (see section below) to build a new flow of the topics - please add dependencies you see currently missing to the list
    - Lars and STRING move forward to introduce networks, but Cytoscape is still at the end of the course.
- Drop Aidan's Day 1 introduction session, begin instead with Marco, who introduces many general topics
- Start the first day with Linux commandline game described by Allegra. We've decided together that we will commit to teaching and using the Linux command line during our exercises, where possible using it in small ways to reinforce and help teach its value
- Swap Day 2 'trainees work with own data' session with Day 4 unseminar session. In this way, they get to do the 'work with their own data' stuff once they've seen more of the differnet tools
    - thus, we will now do the unseminar (Budapest Unseminar in Bioinformatics - BUD?) on Tuesday, and include speed dating as the ice-breaker for this (as we will now begin the course itself with the Linux command line game rather than with speed dating)
    - there will now be three trainee 'work with your own data' sessions, on Thursday, Friday, and Saturday
- Ora S-F gives her keynote during the motif sessions. Therefore the session following (currently 17:00 - 18:30) should be a practical on her peptide docking tools. So that session includes her name (or is just given completely to her). I suppose this provides a modelling dependency too?
- People teaching related topics (see list above) agreed to make sure they work together to reduce redundancy and thus make the time spent teaching the topic less (we're keen to free up some time in the schedule).

Overall, we have to have a balance between depth and breadth so a general aim is to get the students to a useful place from where they can
self-improve their skills.

## Topic dependencies

- PPI databases -> STRING
- Electronic health records -> STRING
- STRING -> Cytoscape
- STRING -> REST